### ISSUE TITLE
1891 Civil Parishes appear to have been incorrectly extracted from TNA information

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/FreeCENMigration#925]
We extracted it as
<subdistrict piece="RG 12/1936"
                         tnaid="C3743381"
                         code="RS 4C"
                         name="Walcot"
                         year="1891">
               <parish name="Bath">
                  <hamlet name="Walcot"/>
               </parish>
The TNA has 
Registration Sub-District 4C Walcot. Civil Parish, Township or Place: Bath: Walcot
Which likely should have been extracted as 2 civil parishes; Bath and Walcot not just Bath when looking at the  census documents

This appears to apply to many records in 1891.

Need to find solution.
Need to assess if same issue applies to other census years

**Comment by richardofsussex** [comment]
From memory, we have interpreted a colon as indicating a subordinate place.  This is certainly what has happened in this case (C3743381): parish = Bath; hamlet = Walcot. Are you saying that TNA have changed the semantics of their free text descriptions for 1891?

**Comment by Captainkirkdawson** [comment]
Have reviewed earlier census years (Just Somerset) and saw no issues of this nature. Reviewed later years and there are no similar issues 

It appears that in 1891 they changed and sometimes stacked civil parishes separated by a colon

@richardofsussex how difficult would it be to reprocess 1891 and treat the an entry following a colon as a new Civil Parish and not as a Hamlet?

If not we will need to add the ability to create a new civil parish for a piece and have people add them by hand.

**Comment by Captainkirkdawson** [comment]
@richardofsussex further research on other counties shows there to be no consistency on civil parish names in 1891 TNA . In other counties there was a second CP after a : but then there was a list of CPs separated by commas !!!
Also there are no Hamlets being recorded in these later census. As they collected more and more information the contents of a piece have become more and more focused on CPs and by 1911 there appear to be almost a 1 for 1
Hence we should likely reprocess 1891 and create what we previously creates as Hamlets as Civil Parishes. 
Bottom line is that it is easier for us to deal with too many CPs rather than too few CPs

**Comment by richardofsussex** [comment]
I suggest that it makes more sense to start from where we have got to, and further process the data without reference to TNA, rather than going back to that source. Essentially, we want to look up all the Hamlets and convert them to Civil Parish if they are one. Given that we can't rely on TNA to infer this, we should use another source. Perhaps this would (finally) be a use for the Administrative Units linked data resource?

**Comment by Captainkirkdawson** [comment]
Thanks @richardofsussex Yes it does appear that they changed their approach for 1891. 
I am happy to update based on what we have.
It is clear that Ancestry have worked from the bottom up rather than the top down ie they started with the Civil Parish extracted from the actual census documents then added the links to ED, sub district and district.
Given that we will make substantive change to the information I now wonder if there is any value to the implementation of a change log.

**Comment by Captainkirkdawson** [comment]
@geoffj-FUG We have converted the civil parishes that were extracted as hamlets in 1891 back to civil parishes. 
You should now be able to process 1891 csv files with CSV Proc.

**Comment by geoffj-FUG** [comment]
Thanks Kirk

 

I will go back to my Walcot file and see what happens.

 

Geoff

**Comment by geoffj-FUG** [comment]
Kirk

 

Bath Walcot was the place that was giving me trouble when we found the problem. It is processed perfectly now and the Civil Parish data looks far better.

 

😊

 

Geoff
